components/brave_rewards/core/database/database_util.cc

```diff
@@ -114,11 +114,7 @@ int GetIntColumn(mojom::DBRecord* record, const int index) {
     return 0;
   }
 
-  if (!record->fields.at(index)->is_int_value()) {
-    DCHECK(false);
-    return 0;
```

Comment:
not sure if these conditions should be removed, but it's up to the wallet team.

Reply:
Well, I'm not sure either, but if we must support failure here, then the DCHECK has to go.

Reply:
I've reached to @zenparsing and he advised to keep the return for the default value, when there's a mismatch. I have just dropped `DCHECK(false)` from the code.